app/app.component.html

```diff
@@ -1,136 +1,114 @@
-<div class="menu-side-bar" (click)="onBodyClick($event)" [hidden]="!isNavigationMenuVisible">
-  <div class="overbody-container"></div>
-  <div class="close-button icon-ic-close" (click)="isNavigationMenuVisible = false"></div>
-  <div class="menu-container table">
-    <div class="table-row">
-      <div class="nav-menu table-cell">
-        <div class="nav-menu-icon-container"><i class="icon-ic-just-arrived-logo"></i></div>
-        <div class="nav-menu-item-container">
-          <div class="nav-menu-item clickable" *ngIf="user">
-            <a class="nav-menu-link" routerLink="/my-jobs">
-              <i class="icon-ic-my-jobs"></i>
-              {{'menu.main.my_assignment' | translate}}
-            </a>
+<div class="navbar-fixed-top" [ngStyle]="(isNavigationMenuVisible || isLanguageMenuVisible) && {'height': '100%'}">
+  <div class="container">
+    <nav class="header navbar navbar-light bg-faded">
+      <div class="row navbar-container">
+        <div class="col-xs row flex-items-xs-middle">
+          <div class="clickable back-button-container" *ngIf="states.length > 1 && currentState !== '/home'" (click)="onBackButtonClick()">
+            <i class="fa fa-angle-left fa-2x back-button"></i>
           </div>
-          <div class="nav-menu-item clickable">
-            <a class="nav-menu-link" routerLink="/jobs/1">
-              <i class="icon-ic-find-jobs"></i>
-              {{'menu.main.find_assignment' | translate}}
-            </a>
-          </div>
-          <div class="nav-menu-item clickable" *ngIf="isCompanyUser">
-            <a class="nav-menu-link" routerLink="/job/create">
-              <i class="icon-ic-create"></i>
-              {{'menu.main.create_assignment' | translate}}
-            </a>
-          </div>
-          <div class="nav-menu-item clickable">
-            <a class="nav-menu-link" href="">
-              <i class="icon-ic-faq"></i>
-              {{'menu.main.faq' | translate}}
-            </a>
-          </div>
-          <div class="nav-menu-item clickable">
-            <a class="nav-menu-link" href="">
-              <i class="icon-ic-contact"></i>
-              {{'menu.main.contact' | translate}}
-            </a>
+          <a routerLink="/home">
+            <img class="logo-img" src="assets/images/logo.png" alt="Just Arrived"/>
+          </a>
+        </div>
+        <div class="col-xs right row flex-items-xs-middle">
+          <div (click)="onLanguageMenuButtonClick()" class="button-container">
+            <div class="nav-language-icon">{{selectedLanguage.languageCode}}<div class="fa lang-arrow" [class.fa-caret-down]="!isLanguageMenuVisible" [class.fa-caret-up]="isLanguageMenuVisible"></div></div>
```

Comment:
Use ngClass to be consistently with the all code in the app.  This syntax [class.fa-caret-down] is look mostly like directive.